### ISSUE TITLE
Albumcovers bij Chromecast

### ISSUE THREAD

**Issue by jeroenbnn** [robinbakker/1tuner#10]
Hi Robin, 

het zou een mooie toevoeging zijn als de albumcovers getoond worden tijdens het afspelen. Vooral wanneer er via de Chromecast gespeeld wordt zou dit leuk zijn.

Wat denk jij hiervan?

--

It would be a nice addition when album covers are showed while listening (sure while listening through Chromecast).

**Comment by robinbakker** [comment]
Ja, zeker weten! Op dit moment is er alleen helaas nog niets voor ingebouwd (muziekinformatie ontbreekt nog). Dus zie je nu alleen nog de radio station of podcast logo's. Eerst moet de algemene Chromecast functionaliteit nog worden afgemaakt, maar helaas nog niet aan toe gekomen...

Bedankt voor je feedback!

--
Yes for sure! Unfortunately, there is nothing built-in for it right now (music information is still missing). So now you only see the logos. The main Chromecast functionality still needs to be completed, but I haven't been able to finish that yet unfortunately. 

Thanks for your feedback!

**Comment by jeroenbnn** [comment]
Super, ik kijk ernaar uit! Wat jij aan het maken bent wilde ik al lang op mijn domeinnaam.
